qutebrowser/app.py

```diff
@@ -496,6 +496,7 @@ def _init_modules(args, crash_handler):
     macros.init()
     # Init backend-specific stuff
     browsertab.init()
+    # pylint: enable=too-many-statements
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.